eng/Versions.props

```diff
@@ -23,11 +23,12 @@
     <ReportGeneratorVersion>4.3.6</ReportGeneratorVersion>
     <!-- Use the correct compiler version -->
     <UsingToolMicrosoftNetCompilers>true</UsingToolMicrosoftNetCompilers>
-    <CodeStyleAnalyersVersion>3.8.0-3.20428.4</CodeStyleAnalyersVersion>
+    <MicrosoftNetCompilersToolsetVersion>3.8.0</MicrosoftNetCompilersToolsetVersion>
```

Comment:
This seems to be a new property but there is no file updated to be using it.

Reply:
This was explicitly removed by @mmitche as this causes a dependency chain issue when inserting analyzers into the SDK repo. We have been asked explicitly to not reference `MicrosoftNetCompilersToolsetVersion`, but instead use in-built compiler toolset from arcade version. Also adding @jmarolf

Reply:
The Microsoft.Net.Compilers.Toolset dependency does not propagate transitively.

Reply:
@sharwell Let me add you to the email thread.